lib/bravo/bill.rb

```diff
@@ -113,6 +113,9 @@ def setup_response(response)
       request_header  = body['FeCAEReq']['FeCabReq']
       request_detail  = body['FeCAEReq']['FeDetReq']['FECAEDetRequest']
 
+      # If there's only one invoice in the batch, put it in an array
+      response_detail = [response_detail] unless response_detail.class == Array
```

Comment:
You could also use `Array()` to wrap the `response_detail` thus avoiding the conditional and the comparation. What do you think?